package.json

```diff
@@ -1,4 +1,16 @@
 {
+  "name": "stac-extensions",
+  "version": "1.0.0",
+  "dependencies": {
+    "remark-cli": "^8.0.0",
+    "remark-lint": "^7.0.0",
+    "remark-lint-no-html": "^2.0.0",
+    "remark-preset-lint-consistent": "^3.0.0",
+    "remark-preset-lint-markdown-style-guide": "^3.0.0",
+    "remark-preset-lint-recommended": "^4.0.0",
+    "remark-validate-links": "^10.0.0",
+    "stac-node-validator": "^1.0.0"
+  },
   "prettier": {
     "printWidth": 80,
```

Comment:
80 is super small, the `@linzjs/style` generally uses `120`

https://github.com/linz/style-js/blob/master/.prettierrc.js#L5

Reply:
120 is way too big - I can't fit two files side by side for comparisons and still be able to read them on a laptop screen. 88 would be fine, and 100 would be just borderline for myself.

Reply:
:shrug: I can get two tabs side by side in vscode on my laptop screen with `120` even the template repo has `127` as its default.

https://github.com/linz/template-python-hello-world/blob/master/pyproject.toml#L6

Reply:
That line length is misleading, unfortunately - I've already reverted pylint to use the default max line length of 88.

Reply:
Oh yeah, I meant to create a PR for that: https://github.com/linz/template-python-hello-world/pull/166